src/components/programmingCard.js

```diff
@@ -0,0 +1,20 @@
+import React from "react"
+import "./programmingCard.scss"
+
+export default function ProgrammingCard({ title, summary, id }) {
+  const href = `https://www.artsmatter.org/programs/${id}`
+  return (
+    <div className="programming-card">
+      <h3 className="programming-card__title">{title}</h3>
```

Comment:
I think anytime we are pulling data from Contentful, we should probably do a check to see if it exists before rendering the element. That way we don't end up with empty elements that could cause spacing issues.

For instance: 

```suggestion
      {title && <h3 className="programming-card__title">{title}</h3>}
```

I also wonder if there is anything we can do to sanitize the data we are displaying, to ensure nothing harmful gets passed in from Contentful? Can you look into this, I haven't experimented with this in Contentful before either, but it might help us prevent the content creator from breaking the site.

Reply:
I will update those components to only display when all of the data is present! I'll also look into sanitizing the data coming in from Contentful, I haven't done this either so I am interested to learn how to go about that with javascript.